lib/fluent/plugin/out_detect_exceptions.rb

```diff
@@ -34,6 +34,8 @@ class DetectExceptionsOutput < Output
     config_param :max_lines, :integer, default: 1000
     desc 'Maximum number of bytes to flush (0 means no limit). Default: 0.'
     config_param :max_bytes, :integer, default: 0
+    desc "Separate log streams by this field (if set). Default: ''."
```

Comment:
Will go with not mentioning the default. This is also consistent with, e.g., remove_tag_prefix.